low-level/framework/base/sspl_constants.py

```diff
@@ -57,18 +57,73 @@
 CONSUL_HOST = consulhost
 CONSUL_PORT = consulport
 SSPL_SETTINGS = {
-        "ACTUATORS" : ["Service", "RAIDactuator", "Smartctl", "NodeHWactuator"],
-        "CORE_PROCESSORS" : ("RabbitMQegressProcessor", "RabbitMQingressProcessor", "LoggingProcessor"),
-        "DEGRADED_STATE_MODULES" : ("ServiceWatchdog", "RAIDsensor", "NodeData", "IEMSensor", "NodeHWsensor",
-                            "DiskMsgHandler", "LoggingMsgHandler", "ServiceMsgHandler", "NodeDataMsgHandler",
-                            "NodeControllerMsgHandler", "SASPortSensor", "MemFaultSensor", "CPUFaultSensor"),
-        "MESSAGE_HANDLERS" : ("DiskMsgHandler", "LoggingMsgHandler", "ServiceMsgHandler", "NodeDataMsgHandler",
-                        "NodeControllerMsgHandler"),
-        "SENSORS" : ["ServiceWatchdog", "RAIDsensor", "NodeData",
-            "IEMSensor", "NodeHWsensor",
-            "SASPortSensor", "MemFaultSensor", "CPUFaultSensor", "RAIDIntegritySensor"]
+        "REALSTORSENSORS": {
```

Comment:
Had given this comment earlier also. 

Having actuators & handles under SENSORS category again doesnt make sense. Why cant we keep something generic, under which its ok to host sensors, actuators, handlers & core processors. One suggestion was 'REALSTORMONITOR'.

Same comment applies elsewhere.

Reply:
These names correspond to pre-existing sections in the conf file.
The new names will also have to be sections.
AIUI these will have only the 'monitor' key.
Is that fine?

Reply:
> These names correspond to pre-existing sections in the conf file.

What removed in this patch(in red), yes they are from existing config, i understand that.

> The new names will also have to be sections. AIUI these will have only the 'monitor' key. Is that fine?

I dont think we need so many changes to this patch further, just rename 'REALSTORSENSORS' to 'REALSTORMONITOR' on line 60. Same applies to other sections

Reply:
> These names correspond to pre-existing sections in the conf file.
All the new top-level key names (`REALSTORSENSORS`, `REALSTORSENSORS`, etc.) correspond to sections in the config.
They must be changed in-sync with the names of sections themselves.

Reply:
Ok so REALSTORMONITOR will have to section names you mean to address this comment. In that case, no we wont change that this time. Ultimately we need to later though, can you create a task tkt to address in LDR R2